resources/js/app.js

```diff
@@ -1,25 +1,31 @@
 require('./bootstrap');
 
 window.Vue = require('vue');
-import Bugsnag from '@bugsnag/js'
-import BugsnagPluginVue from '@bugsnag/plugin-vue'
+import * as Sentry from "@sentry/vue";
+import { Integrations } from "@sentry/tracing";
 
 Vue.component('kiosk', require('./components/Kiosk.vue').default);
 Vue.component('home-page', require('./components/HomePage.vue').default);
 
 import VueSweetalert2 from 'vue-sweetalert2';
 Vue.use(VueSweetalert2);
 
-if (process.env.MIX_BUGSNAG_API_KEY !== undefined) {
-    Bugsnag.start({
-        apiKey: process.env.MIX_BUGSNAG_API_KEY,
-        plugins: [new BugsnagPluginVue()],
-        releaseStage: process.env.MIX_APP_ENV
-    })
-    Bugsnag.getPlugin('vue')
-        .installVueErrorHandler(Vue)
+if (process.env.MIX_SENTRY_DSN !== undefined) {
+    Sentry.init({
+        Vue: Vue,
+        dsn: process.env.MIX_SENTRY_DSN,
+        environment: process.env.MIX_APP_ENV,
+        attachProps: true,
+        logErrors: true,
+        integrations: [new Integrations.BrowserTracing()],
+        tracesSampleRate: 1.0,
```

Comment:
I'm not sure how many requests we get (probably pretty low) but just in case we shouldn't set this to 1

```suggestion
        tracesSampleRate: .5,
```

Reply:
I would keep it at 1. We have very low request volumes.